creating_images/index.adoc

```diff
@@ -6,3 +6,7 @@
 :experimental:
 
 This guide provides best practices on writing and testing Docker images that can be used on OpenShift.
+ifdef::openshift-enterprise,openshift-origin[]
```

Comment:
@pweil-
The conditional is around a link to the Installation and Configuration document, which is OSO and OSE only, IIUC its [`_build_cfg.yml` Distros key](https://github.com/openshift/openshift-docs/blob/master/_build_cfg.yml#L152) correctly.
Am i missing something?